File: week11/week11-2.py
bound=20000
table=[True]*bound
ans=0
for i in range(2,bound):
  if table[i]==True:
    #print(i,end=' ')
    ans+=1
    for k in range(i*i,bound,i):
      table[k]=False
print('質數有:',ans,'個')

File: week11/week11-2.cpp
///week11-2
# include <stdio.h>
int main(){
    int bound=20000,ans=0;
    int table[20000]={};
    for(int i=2;i<bound;i++){
        if(table[i]==0){
            ans++;
            for(int k=i*i;k<bound;k+=i){
                table[k]=-1;
            }
        }
    }
    printf("½è¼Æ¦³: %d\n",ans);
}
